app/controllers/courses_controller.rb

```diff
@@ -121,7 +121,7 @@ def save_deadlines
   end
 
   def enable
-    authorize! :teach, @organization
+    authorize! :teach, @organization # should assistants be able to enable/disable?
```

Comment:
There is probably not enough time in this project but ideally we'd have more fine-grained abilities and allow organization admins to delegate them selectively.